tests/bud.bats

```diff
@@ -2837,6 +2837,25 @@ EOF
   expect_output --substring "\[2/2] STEP 1/2: FROM busybox"
 }
 
+# Following test must fail since we are trying to run linux/arm64 on linux/amd64
+# Issue: https://github.com/containers/buildah/issues/3712
```

Comment:
Note that such hard-coded requirements to test platform nail it down to only run on amd64.

If there's an alternative, e.g. to specify `linux/bogus` and check for a specific error message, I'd prefer that.